grafana/tasks/config.yml

```diff
@@ -1,12 +1,21 @@
 ---
-- name: copy startup scripts
-  template: src="{{ item.src }}" dest="{{ item.dest }}" owner="root" group="root" mode='0755'
-  with_items:
-    - { src: 'grafana.sh.j2', dest: "/etc/init.d/grafana" }
+- name: Copy Granafa initd script
```

Comment:
nitpick in config name: Granafa -> Grafana

Reply:
Not the first time I misspelled Grafana - nice catch!